### PR TITLE
improve: better errors when crashing

### PR DESCRIPTION
**Pull request by FelixAbrahamsson**

### CODE REVIEW

**Review by samedii — APPROVED** [review]

File: kmm/positions/read_kmm.py [hunk 1]
```diff
@@ -6,32 +6,35 @@
...
+            dtype=dict(
+                track_section=str,
+                kilometer=np.int32,
+                meter=np.int32,
+                track_lane=str,
+                sweref99_tm_x=np.float32,
+                sweref99_tm_y=np.float32,
+            ),
+        )
+    except Exception:
```
Comment (Jim-Holmstroem) [inline kmm/positions/read_kmm.py]:
```suggestion
    except Exception as e:
```

File: kmm/positions/read_kmm.py [hunk 2]
```diff
@@ -6,32 +6,35 @@
...
+        raise ValueError("Unable to parse kmm2 file, invalid csv.")
```
Comment (Jim-Holmstroem) [inline kmm/positions/read_kmm.py]:
```suggestion
        raise ValueError("Unable to parse kmm2 file, invalid csv.") from e
```

Comment (Jim-Holmstroem) [reply kmm/positions/read_kmm.py]: to re raise the exception and give it a cause

```
ParseError: Too many columns specified
The above exception was the direct cause of the following exception:
ValueError: Unable to parse kmm2 file
```

Comment (Jim-Holmstroem) [reply kmm/positions/read_kmm.py]: https://docs.python.org/3/tutorial/errors.html#exception-chaining

Comment (FelixAbrahamsson) [reply kmm/positions/read_kmm.py]: :open_mouth: very nice, thanks for the tip

**Pull request merged**